src/main/java/seedu/address/logic/parser/task/TaskCliSyntax.java

```diff
@@ -0,0 +1,15 @@
+package seedu.address.logic.parser.task;
+
+import seedu.address.logic.parser.Prefix;
+
+/**
+ * Contains Command Line Interface (CLI) syntax definitions common to multiple task-related commands
+ */
+public class TaskCliSyntax {
+
+    /* Prefix definitions */
+    public static final Prefix PREFIX_DESCRIPTION = new Prefix("d/");
```

Comment:
i think this prefix is already used for periodOfStay in PatientCliSyntax.java, will it be affected?

Reply:
Nope, it is in an entirely different file. That said, I wonder if it will be confusing for the user?

Reply:
i think its fine. It makes me wonder if I should convert my editroom prefixes to p/ instead of pn/ for patient name. hahahah

Reply:
I think `p/` is good 😄 We can try to keep it as short and sweet as possible since that's easier to remember.